pyproject.toml

```diff
@@ -7,7 +7,7 @@ name            = "psydac"
 version         = "0.1"
 description     = "Python package for isogeometric analysis (IGA)"
 readme          = "README.md"
-requires-python = ">= 3.8, < 3.12"
+requires-python = ">= 3.9" #, < 3.12"
```

Comment:
I think it is better to set an upper bound, because we should guarantee that the versions specified here do work.

Reply:
indeed 👍